tests/integration_tests/functional/test_snapshot_basic.py

```diff
@@ -414,3 +414,164 @@ def test_cmp_full_and_first_diff_mem(network_config,
                              ])
 
     test_matrix.run_test(_test_compare_mem_files)
+
+
+def test_negative_snapshot_lifecycle(bin_cloner_path):
```

Comment:
This test is not necessarily related to snapshots, but rather to the microvm's lifecycle in general. I would move this to `test_api.py`.

Reply:
👍  this test doesn't belong here.